web/templates/homepage/index.html.eex

```diff
@@ -59,18 +59,21 @@
   <% end %>
 </section>
 
-<section class="pa4 tc-ns pa1 ph3 pb4 pb5-l" id="suggestion_form">
-  <div class="br1 shadow-2 ph3 pt3 pb2 mt4 tl w-60-ns center">
+<section class="pa4 tc-ns tl-m pa1 ph3 ph5-m ph6-l pb4 pb5-l w-100" id="suggestion_form">
```

Comment:
The text needs to align with the text at the very top, and the box needs to align with that